frappe/database/database.py

```diff
@@ -588,12 +588,6 @@ def get_values(
 					if ignore and (frappe.db.is_missing_column(e) or frappe.db.is_table_missing(e)):
 						# table or column not found, return None
 						out = None
-					elif (not ignore) and frappe.db.is_table_missing(e):
-						# table not found, look in singles
-						out = self.get_values_from_single(
```

Comment:
This fallback is more than just single value fallback, this also made `ignore` argument useless against missing tables :facepalm: